README.md

```diff
@@ -206,6 +206,8 @@ Either way it provides access to the following fields:
 * **runAsGroup** The group ID to run all containers in the pod as. 
 * **hostNetwork** Use the hosts network.
 * **workspaceVolume** The type of volume to use for the workspace. Can be `emptyDirWorkspaceVolume` (default), `dynamicPVC()`, `hostPathWorkspaceVolume()`, `nfsWorkspaceVolume()`, or `persistentVolumeClaimWorkspaceVolume()`.
+* **retryOnQuotaViolation** Retry queued build if kubernetes reports a resource quota exceeded or in conflict. Defaults to `true`
```

Comment:
I don't think you need a new option. Exceeding quota is a valid case for retry.

Reply:
I've removed it as a configuration option.